packages/terra-search-field/src/SearchField.jsx

```diff
@@ -86,6 +92,7 @@ const defaultProps = {
   placeholder: '',
   searchDelay: 250,
   value: undefined,
+  inputAttributes: undefined,
```

Comment:
@StephenEsser Should we provide a default for this? I don't remember what we had decided

Reply:
We discussed re-enabling the lint rule that enforces any non-required props to have a default so we should probably move forward with providing defaults. Eventually re-enabling this rule to enforce it. 

https://github.com/cerner/eslint-config-terra/blob/master/eslint.config.js#L15

Reply:
Should this be {} then since we are expecting an obj ?

Reply:
My preference is to default to `undefined`, but there is no hard rule on this and there are cases of both throughout the library.

I prefer to default to `undefined` so the internal logic for non-require props is always identical to what the consumer has declared and it works smoother with utilities like redux where there may or may not be an object.

Reply:
What does Object.assign do if inputAttributes is undefined? Does it make a new hash, or does it crash? If it crashes, I strongly suggest defaulting to {}, as we shouldn't require the developer to pass in any options if they don't need any

Reply:
I tested it, and it looks like undefined creates the blank hash just fine.